components/suggestion/suggestion_list.jsx

```diff
@@ -192,40 +191,39 @@ export default class SuggestionList extends React.PureComponent {
     }
 
     render() {
-        if (!this.props.open || this.props.cleared) {
+        const {
+            items,
+            terms,
+            selection,
+            components,
+            open,
+            cleared,
+            renderDividers,
+            renderNoResults,
+        } = this.props;
+
+        if (!open || cleared) {
             return null;
         }
 
-        const clonedItems = cloneDeep(this.props.items);
-
-        const items = [];
-        if (clonedItems.length === 0) {
-            if (!this.props.renderNoResults) {
+        const handledItems = [];
+        if (items.length === 0) {
+            if (!renderNoResults) {
                 return null;
             }
-            items.push(this.renderNoResults());
+            handledItems.push(this.renderNoResults());
         }
 
-        const sortedItems = clonedItems.sort((itemA, itemB) => {
```

Comment:
Should we remove the sorting logic, Is it redundant because I can see couple of components either using or extending ``suggestion_list``, Not sure if they rely on this for ordering. Thanks